domain/schema/model.go

```diff
@@ -131,13 +137,14 @@ INSERT INTO change_log_namespace VALUES
     (6, 'storage_volume', 'Storage volume changes based on UUID'),
     (7, 'storage_volume_attachment', 'Volume attachment changes based on UUID'),
     (8, 'storage_volume_attachment_plan', 'Volume attachment plan changes based on UUID'),
-    (9, 'secret', 'Secret auto prune changes based on UUID'),
+    (9, 'secret_metadata', 'Secret auto prune changes based on UUID'),
```

Comment:
The namespace should be "secret_auto_prune" because that's what the change represents , right?
The trigger is specifically a change to the auto_prune column.